test/unit/core/plugins/json-schema-2020-12-samples/fn.js

```diff
@@ -2983,3 +2984,55 @@ describe("memoizedCreateXMLExample", () => {
     ).toEqual(updatedExpected)
   })
 })
+
+describe("mergeJsonSchema", function () {
```

Comment:
```suggestion
describe("merge", function () {
```

Here the function is actually called `merge`, but it's exposed as `mergeJsonSchema`. Let's use it's real name in tests.